.github/workflows/ci.yaml

```diff
@@ -29,3 +29,50 @@ jobs:
         run: |
           flit install -s
           pytest tests/unit
+
+  linux-integration_tests-sqlserver:
+    name: "Linux - integration tests - Python ${{ matrix.PYTHON_VERSION }} - SQL server with ${{ matrix.DRIVER }} driver"
+    runs-on: ubuntu-20.04
+    env:
+      CI: True
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION: ['3.8']
+        DRIVER: ["mssql"] #, "mssql-freetds"]

```

Comment:
Would we like to test for both? If not, we might be able to remove the comment.

Reply:
You're right, will add the other driver (hopefully it works).